act-rules-format.md

```diff
@@ -497,6 +523,65 @@ Background (optional) {#background}
+
+In addition to `passed`, `failed`, and `inapplicable`, implementations can report `cantTell` and `untested` as an [=outcome=]. If no outcome is given for a test case, it is considered `untested`.
+
+There are three levels of consistency; complete, partial, and minimal. Consistency levels are defined in the following way. The consistency of a set of implementations procedures is the first level in the following list that matches the definition.
+
+1. <dfn>Complete</dfn> implementations consider all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `failed` test cases are reported as `failed`, or `cantTell` by at least one procedure; and
+    - All `passed` and `inapplicable` test cases are reported as `passed`, or `inapplicable`, or `cantTell` by all procedures; and
+    - `passed` and `failed` test cases are not all reported as `cantTell` by all procedures.
+    - the implementation procedures report a [=consistent accessibility requirements mapping=].
+
+2. <dfn>Partial</dfn> implementations consider some, but not all aspects of a rule's applicability and expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `passed` and `inapplicable` test cases are reported as `passed`, `inapplicable`, `cantTell`, or `untested`; and
+    - `passed` and `failed` test cases are not all reported as `cantTell` or `untested`.
+
+3. <dfn>Minimal</dfn> implementations consider some or all aspects of a rule's applicability, without considering its expectations. For the [=outcomes=] reported for some set of [=implementation procedures=], all the following are true:
+
+    - All `passed` and `inapplicable` test cases are reported as `passed`, `inapplicable`, `cantTell`, or `untested`; and
+    - `inapplicable` test cases are not all reported as `cantTell` or `untested`.
+
+
+### Coverage ### {#impl-coverage}
+
+Not all implementations are able to test all aspects of an ACT Rule. Particularly in automation, it is common for edge cases to be reported as `cantTell`. Coverage is a measure of how much of an ACT Rule is tested by an implementation. Similar to [consistency](#impl-consistency), the coverage of an ACT Rule is determined for a set of [=implementation procedures=]. Coverage is expressed using two numbers; total coverage, and automated coverage.
+
+The <dfn>total coverage</dfn> is expressed as a fraction where the denominator is the number of test cases, and the numerator is the number of test cases for which there was at least one correct outcome. `failed` outcomes are correct if it is for a `failed` test case. `passed` and `inapplicable` outcomes are correct if they are for a `passed` or `inapplicable` test case. Other outcomes are not considered correct.
```

Comment:
This is wrong; A passed test case that is reported with both a failed and passed outcome is not correct. Need to look at how to rephrase this.